integration_tests/test_batch_jobs.py

```diff
@@ -0,0 +1,25 @@
+from .rest_api_utils import (
+    CREATE_BATCH_JOB_REQUEST,
+    CREATE_DOCKER_IMAGE_BATCH_JOB_BUNDLE_REQUEST,
+    CREATE_DOCKER_IMAGE_BATCH_JOB_REQUEST,
+    USER_ID_0,
+    cancel_batch_job,
+    create_batch_job,
+    create_docker_image_batch_job,
+    get_or_create_docker_image_batch_job_bundle,
+)
+from .test_bundles import model_bundles  # noqa
+
+
+def test_di_batch_jobs(model_bundles) -> None:  # noqa
```

Comment:
If we're skipping this, should we do `pytest.mark.skip`? (does this run via `pytest`?)